### DISCUSSION TITLE
Pacakaging with Pyinstaller

### DISCUSSION THREAD

**Discussion by koukii007** [TomSchimansky/CustomTkinter#730] [General]
Hi
thank you for this great module it helped me create a very nice program.
But i am having issues packaging my program using pyinstaller an auto-py-to-exe. 
I keep receiving errors like this one
![image](https://user-images.githubusercontent.com/91621764/201330607-34867e1a-bf16-465c-b34a-8c0592d22ac8.png)
I used to have an error saying **No module found : darkdetect** I solved that by following the wiki page . I got rid of the attached error by commenting out the lines importing that module since my program was still working but then it gave me another modult not found error.

AM open to any suggestions
thank you

**Comment by Evolution0** [comment]
Seems like a pyinstaller issue to me, auto-py-to-exe is kind of a frontend for it.

Check [here](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/?utm_source=auto_py_to_exe&utm_medium=readme_link&utm_campaign=auto_py_to_exe_help#modulenotfounderror-no-module-named-x-importerror-no-module-named-x) about missing modules.

If you remove `darkdetect` I imagine you won't have light/dark mode automatic switching so if your theme is set to system it might error out or crash entirely if the system theme changes while the program is running.

Also its having issues finding all required modules likely, it simply moved to the next alphabetical one to throw the same error.

**Comment by koukii007** [comment]
You are right it's going alphabetically.
I will check out the link you sent
Thank you

**Comment by Rezooo86** [comment]
I'm not sure if it's the same issue be I had the following error message when trying to open the exe file created with pyinstaller ==> the same file was working properly when using Tkinter instead of customtkinter   Any help !!!!!
![image](https://user-images.githubusercontent.com/66188638/205480466-7072966e-d17c-4046-aa1f-3fb879c46fee.png)
